src/protocolgame.cpp

```diff
@@ -793,8 +793,10 @@ void ProtocolGame::parsePacket(NetworkMessage& msg)
 			// case 0xFE: break; // store window history 2
 
 		default:
+			// we cannot pass an unique_ptr as capture here because
+			// std::function requires the callable object to be *copyable*
 			g_dispatcher.addTask([=, playerID = player->getID(), msg = new NetworkMessage(msg)]() {
```

Comment:
```cpp
g_dispatcher.addTask([=, playerID = player->getID(), msg = std::make_unique<NetworkMessage>(msg)]() {
```

Reply:
You can't do that, `std::function` **requires** the object to be **copyable**, and `std::unique_ptr` is not copyable.